Tools/environment_install/install-prereqs-ubuntu.sh

```diff
@@ -111,20 +112,22 @@ fi
 SCRIPT_DIR=$(dirname $(realpath ${BASH_SOURCE[0]}))
 ARDUPILOT_ROOT=$(realpath "$SCRIPT_DIR/../../")
 
+HOMEDIR=`sudo su - $USER -c "echo ~"`
```

Comment:
$HOME or /home/$USER directly ? no need for using sudo su

Reply:
Please see https://github.com/ArduPilot/ardupilot/issues/12513

In this case `/home/` is a magic string. In Docker the home folder is set to `/ardupilot/` and could be arbitrary. And in this case, Docker runs this script as `root`.